packages/web/src/form/form.js

```diff
@@ -35,6 +35,12 @@ const INPUT_TYPES = [
   'url',
   'week',
 ]
+const COERCION_FUNCTIONS = {
+  Boolean: (value) => !!value,
+  NumberField: (value) => parseInt(value, 10),
+  Int: (value) => parseInt(value, 10),
+  Float: (value) => parseFloat(value),
```

Comment:
In Sweden we use comma (,) as our decimal separator, which parseFloat doesn't support. It only supports dot as a decimal separator. Is this good enough?

Reply:
I would think so? Once we look into Internationalization and Localization maybe we can do that automatically for people if they set their locale to somewhere that uses a comma for a decimal separator.

Reply:
Great, I'll leave it as-is